src/ZenithAngleCalc.jl

```diff
@@ -17,18 +17,18 @@ end
 # calculate the distance, declination, and hour angle (at lon=0)
 function distance_declination_hourangle(::Type{FT},
                                         date::DateTime,
-                                        param_set::APS,
+                                        param_set::IP.APS,
                                         eot_correction::Bool,
                                         milankovitch::Bool) where {FT <: Real}
-    Ya::FT = year_anom(param_set)
-    day_length::FT = Planet.day(param_set)
-    d0::FT = orbit_semimaj(param_set)
-    M0::FT = mean_anom_epoch(param_set)
-    
+    Ya::FT = IP.year_anom(param_set)
+    day_length::FT = IP.day(param_set)
+    d0::FT = IP.orbit_semimaj(param_set)
+    M0::FT = IP.mean_anom_epoch(param_set)
+
     epoch_string = "2000-01-01T11:58:56.816"
```

Comment:
This is where `epoch_string` is currently defined which I know is not ideal.

Reply:
Let's tackle this in a later PR, since this one is already pretty busy.